pages/apply.html

```diff
@@ -11,52 +11,101 @@
   <link rel="stylesheet" type="text/css" href="../src/css/index.css">
   <link rel="stylesheet" type="text/css" href="../src/css/grid.css">
   <link rel="stylesheet" type="text/css" href="../src/css/menu.css">
+
+  <!--Bootstrap-->
+  <link rel="stylesheet" type="text/css" href="../assets/vendor/bootstrap.css">
+
+  <!--Bootstrap js, Popper.js and jQuery -->
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+    integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo"
+    crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
+    integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
+    crossorigin="anonymous"></script>
+
+  <style>
```

Comment:
Can you create a new css file for these styles?

Reply:
Those are the bootstrap styles right? Do you mean saving a copy of those bootstrap files in our local folder instead of calling them in this line?

Reply:
No specifically these custom inline css in the html

Reply:
these are fixed now, moved to `apply.css`